piper_bringup/launch/piper.launch.xml

```diff
@@ -22,4 +22,13 @@
           args="joint_trajectory_controller gripper_controller joint_state_broadcaster -c /controller_manager" />
 
     <node pkg="rviz2" exec="rviz2" args="-d $(find-pkg-share piper_bringup)/rviz/robotmodel.rviz" output="both"/>
+
+    <!-- official python CAN driver -->
+    <node pkg="piper" exec="piper_single_ctrl" output="both">
```

Comment:
Only need to run this when `mock_hardware=false`

Reply:
good point! I added that locally already